.github/workflows/release_tag.yml

```diff
@@ -0,0 +1,135 @@
+name: Create Release for tag
+
+on:
+  push:
+    tags:
+      - 'v*'
+      - '!v*-alpha*'
+      - '!v*-beta*'
+      - '!v*-rc*'
+jobs:
+  provide_changed_packages:
+    # see json juggling: https://docs.github.com/en/free-pro-team@latest/actions/reference/context-and-expression-syntax-for-github-actions#example-6
+    # see https://stackoverflow.com/a/62953566/1348344
+    runs-on: ubuntu-latest
+
+    steps:
+      - name: Set Environment
+        run: |
+          echo "BUILD_TAG=${GITHUB_REF/refs\/tags\//}" >> $GITHUB_ENV
+
+      - uses: actions/checkout@v2
+
+      # load current composer.lock
+      - id: currentLock
+        name: Gather latest package information
+        run: |
+          OUT=$(jq --slurp '[.[].packages[] | select(.name | contains("ezsystems") or contains("ibexa")) | [.name, .version] | { name: (.[0]), version: .[1] }]' composer.lock)
+          echo "::set-output name=lock::$( echo "$OUT" | sed ':a;N;$!ba;s/\n/%0A/g' )"
+
+      - name: Get previous release tag based on type
+        id: prevrelease
+        uses: ibexa/version-logic-action@master
+        with:
+          currentTag: ${{ env.BUILD_TAG }}
+
+      # checkout previous tag
+      - uses: actions/checkout@v2
+        with:
+          ref: ${{ steps.prevrelease.outputs.previousTag }}
+
+      # load previous composer.lock
+      - id: previousLock
+        name: Gather previous package information
+        run: |
+          OUT=$(jq --slurp '[.[].packages[] | select(.name | contains("ezsystems") or contains("ibexa")) | [.name, .version] | { name: (.[0]), version: .[1] }]' composer.lock)
+          echo "::set-output name=lock::$( echo "$OUT" | sed ':a;N;$!ba;s/\n/%0A/g' )"
+
+      # do some magic comparing those outputs
+      - id: output_data
+        name: Do comparison and output JSON with changes
+        run: |
+          FILE1=$(mktemp)
+          FILE2=$(mktemp)
```

Comment:
Is there any change to name it more meaningful?

Reply:
There's no need as those are just technical files.